docs/src/Module.doc.js

```diff
@@ -36,6 +36,23 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'expandedId',
```

Comment:
Since the parent component has knowledge of the `items` array passed in, there's no need for an id here. It's simpler to just use the index. So let's change this to `expandedIndex`, an optional number.